File: BankingService/src/main/java/com/bankibgapp/bankingservice/model/dto/UserDto.java
package com.bankibgapp.bankingservice.model.dto;

import lombok.Data;

import java.util.List;
import java.util.UUID;

@Data
public class UserDto {

    private UUID id;
    private String firstName;
    private String lastName;
    private String email;
    private String identificationNumber;
    private List<BankAccountDto> bankAccounts;
}

File: BankingService/src/main/java/com/bankibgapp/bankingservice/model/dto/UserDTO.kt
package com.bankibgapp.bankingservice.model.dto

import java.util.UUID

data class UserDTO @JvmOverloads constructor(
    val id:UUID?=null,
    val firstName:String?=null,
    val lastName:String?=null,
    val email:String?=null,
    val identificationNumber:String?=null,
    val bankAccounts: List<BankAccountDTO>?=null

){
}